File: 905-sort-array-by-parity/905-sort-array-by-parity.py
class Solution:
    def sortArrayByParity(self, nums: List[int]) -> List[int]:
        arr = []
        newarr = []
        for element in nums:
            if element % 2 == 1:
                arr.append(element)
            else:
                newarr.append(element)
        newarr.extend(arr)
        return newarr

File: 905-sort-array-by-parity/905-sort-array-by-parity.java
class Solution {
    public int[] sortArrayByParity(int[] nums) {
        int arr[]=new int [nums.length];
        int j=0;
        int k=nums.length-1;
        for(int i=0;i<nums.length;i++){
            if(nums[i]%2==0){
                arr[j]=nums[i];
                j++;
            }else{
                arr[k]=nums[i];
                k--;
            }
        }
        return arr;   
    }
}